app/liquid_tags/codesandbox_tag.rb

```diff
@@ -14,18 +15,39 @@ def render(_context)
   private
 
   def parse_id(input)
-    input_no_space = input.delete(" ")
-    if valid_id?(input_no_space)
-      input_no_space
-    else
-      raise StandardError, "Invalid codesandbox Id"
+    id = input.split(" ").first
+    raise StandardError, "Invalid codesandbox ID" unless valid_id?(id)
 
-    end
+    id
   end
 
   def valid_id?(id)
     id =~ /\A[a-zA-Z0-9\-]{0,60}\Z/
   end
+
+  def parse_options(input)
+    _, *options = input.split(" ")
+
+    # Validation
+    validated_options = options.map { |o| valid_option(o) }.reject { |e| e == nil }
+    raise StandardError, "Invalid Options" unless options.empty? || !validated_options.empty?
+
+    query = options.join("&")
```

Comment:
I think you meant `validated_options`?
```suggestion
    query = validated_options.join("&")
```